app/src/main/java/com/example/bigowlapp/activity/SearchContactsByPhone.java

```diff
@@ -114,6 +114,7 @@ public void onComplete(@NonNull Task<QuerySnapshot> task) {
                                     else {
                                         Toast.makeText(SearchContactsByPhone.this, "User doesn't have the app", Toast.LENGTH_SHORT).show();
                                         Intent intent = new Intent(SearchContactsByPhone.this, SendSmsInvitationActivity.class);
+                                        intent.putExtra("number1", number2);
```

Comment:
same with intent extra ID, should be more descriptive.

Reply:
ok. Changes have been done

Reply:
thanks